Directory.Build.props

```diff
@@ -111,6 +111,7 @@
   <PropertyGroup>
     <!-- Ensure API docs are available. -->
     <NoWarn>$(NoWarn.Replace('1591', ''))</NoWarn>
+    <NoWarn>$(NoWarn);0105</NoWarn>
```

Comment:
If these warnings are specific to VS scenarios, could this be conditional on '$(BuildingInsideVisualStudio)' == 'true'❔

@Tratcher mentioned in https://github.com/dotnet/aspnetcore/pull/34324#discussion_r669853363 seeing numerous errors when using Middleware.slnf and that makes me concerned about a global suppression versus targeted use of things like `<Import Remove="System.Net.Http" />` or `<DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>` as @JunTaoLuo did in #34236 (where the implicit `using`s first came into this repo). Why not fix the duplications instead of suppressing warnings about them❔

Reply:
The immediate goal is to unblock contributors trying to use VS2019. Going through the repo and cleaning out all of the duplicates is a longer task that we can divide up and accomplish at our leisure. Explicit `Import Remove`'s have a similar problem, we'd have to test all of the projects to figure out which duplicates to remove.

I'll try out the BuildingInsideVisualStudio check.

Reply:
Sounds good. Much appreciated.